lib/mp4/transmuxer.js

```diff
@@ -21,8 +21,8 @@ var AdtsStream = require('../codecs/adts.js');
 var H264Stream = require('../codecs/h264').H264Stream;
 var AacStream = require('../aac');
 var isLikelyAacData = require('../aac/utils').isLikelyAacData;
+var ONE_SECOND_IN_TS = require('../utils/clock').ONE_SECOND_IN_TS;
```

Comment:
This isn't a required change but we use it this way in most other files. I think I missed this during the original rebase.